src/main/java/com/duke/modules/Storage.java

```diff
@@ -0,0 +1,121 @@
+package com.duke.modules;
+
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+import java.util.ArrayList;
+
+import com.duke.tasks.Deadline;
+import com.duke.tasks.Event;
+import com.duke.tasks.Task;
+import com.duke.tasks.Todo;
+
+/**
+ * A class responsible for saving/loading a task list into/from a text file.
+ */
+public class Storage {
+    private static Storage STORAGE_INSTANCE;
+    private String directoryPath = System.getProperty("user.dir") + "/data/";
+    private String listFile = "listData.txt";

```

Comment:
if planning to keep the name of the file constant, can consider using uppercase for listFile